ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/DX9screen.cpp

```diff
@@ -20,6 +20,7 @@
 #include "Bridges/General/DX9Context.h"
 #include "Direct3D9Headers.h"
 #include "../General/GSbackground.h"
+#include "../General/GStiles.h"
```

Comment:
Please don't use ../ in includes it makes it a pain to move anything later

Reply:
Correct, but it's in a million places in each system, I don't know if maybe I should just do that in a separate pr?

Reply:
Imo just remove them when you see them. I already made huge pr to remove all the ../../ includes but you introduced a bunch again when I waasn't looking

Reply:
I thnk im in the blame of every header file lol

Reply:
Tbh, iirc, you did not touch graphics, because I haven't since. I know I'm responsible for a lot of these, but if I added more I probably just forgot and went with the convention the file already had. But ok, I can redo all the files I've changed. There's no need to add the General folder to the makefile is there? I think when me and Josh set these up we must have already done that because it has no problems finding them.

Reply:
The comit im tralking about was before you made bridges but I removed literally every ../

Reply:
Alright, I've made the change using rectangle select for all 4 of the screen files in e58e04029efe701ae239a32e5ced14b1d4885bea